digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -848,6 +883,7 @@ Returns empty result as Json message, not as page not found
   }
 
   private def validateUserMunicipalityAccess(user: User)(municipality: Int): Unit = {
+    if (!user.isServiceRoadMaintainer())

```

Comment:
This IF doesn't do nothing.
The idea was to put the halt(Unauthorized("ServiceRoad user is only authorized to alter serviceroad assets")) like was placed in the others IF's?